apps/tsilhqotin-language-hub/src/Components/Pages/Funders/Funders.tsx

```diff
@@ -14,9 +16,16 @@ export function Funders(props: FundersProps) {
                     <h1 id="pageTitle">Funders</h1>
```

Comment:
The funders data needs to be read from a config as well. Now is the time to do that. It makes this maintainable in the future. You simply manage a json file and the system "fails fast" if you make a mistake with your updates. Theses changes to the data will show up immediately.